pkg/providers/nutanix/template.go

```diff
@@ -158,7 +159,14 @@ func buildTemplateMapCP(
 	kubeletExtraArgs := clusterapi.SecureTlsCipherSuitesExtraArgs().
 		Append(clusterapi.ResolvConfExtraArgs(clusterSpec.Cluster.Spec.ClusterNetwork.DNS.ResolvConf)).
 		Append(clusterapi.ControlPlaneNodeLabelsExtraArgs(clusterSpec.Cluster.Spec.ControlPlaneConfiguration))
+
+	auditPolicy, err := common.GetAuditPolicy(clusterSpec.Cluster.Spec.KubernetesVersion)
```

Comment:
Just curious where this policy comes from. Should it be updated with latest from upstream?
https://github.com/kubernetes/kubernetes/blob/v1.29.0/cluster/gce/gci/configure-helper.sh#L1144

Maybe have it be dynamic based on the minor k8s version?

Reply:
you are right it should be updated with upstream. we haven't done that yet.